main/views.py

```diff
@@ -82,11 +84,12 @@ class UserProfileView(LoginRequiredMixin, TemplateView):
     profile_form = UserProfileForm
 
     def get(self, request, *args, **kwargs):
+        experiences = models.Experience.objects.all()
         user_form = self.user_form
         profile_form = self.profile_form
 
         return render(request, self.template_name,
-                      {'u_form': user_form, 'p_form': profile_form})
+                      {'u_form': user_form, 'p_form': profile_form, 'exp': experiences})
```

Comment:
always use a correct naming conventions, instead of 'exp': experiences use 'experiences_list': experiences or 'experiences_list': experiences_list